ModPatches/ABF Synstruct Biomimetics/Patches/ABF Synstruct Biomimetics/Bodies/Bodies_Animal_Airborne.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_Razorstruct"]/corePart/groups</xpath>
+		<value>
+			<li>CoveredByNaturalArmor</li>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/BodyDef[defName="ABF_Body_Synstruct_Razorstruct"]/corePart/parts/li[def="ABF_BodyPart_Synstruct_Torso"]/groups</xpath>
```

Comment:
non-existing patch path, natural armor to torso is applied by previous operation